recipe/meta.yaml

```
@@ -1,55 +1,58 @@
 {% set name = "wandb" %}
 {% set version = "0.10.15" %}
 
+
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wandb-{{ version }}.tar.gz
   sha256: 1cd7f77c4db81e1f8e93785d741155b3017711b64e08de9d76895514ed32d192
 
 build:
-  number: 0
+  number: 1
   noarch: python
   entry_points:
-    - wandb=wandb.cli:cli
-    - wb=wandb.cli:cli
-    - wanbd=wandb.cli:cli
-    - wandb-docker-run=wandb.cli:docker_run
+    - wandb=wandb.cli.cli:cli
+    - wb=wandb.cli.cli:cli
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
-    - python
+    - python >=3.6
   run:
     - click >=7.0
     - configparser >=3.8.1
     - docker-pycreds >=0.4.0
     - gitpython >=1.0.0
-    - gql
-    - nvidia-ml
+    - promise >=2.0,<3
```

Perhaps these should be re-added?
```suggestion
    - gql
    - nvidia-ml
    - promise >=2.0,<3
```

---

Thoughts @conda-forge/wandb? I'm not in a position to know either way I don't know anything about this package but this PR is needed to fix #17.

---

Thoughts @conda-forge/wandb? I'm not in a position to know either way I don't know anything about this package but this PR is needed to fix #17.